--- a/Makefile
+++ b/Makefile
@@ -59,13 +59,10 @@
 install: all
 	install -d $(DESTDIR)/usr/sbin
 	install -d $(DESTDIR)/etc
-	install -d $(DESTDIR)/lib/systemd/system
-	install -d $(DESTDIR)/usr/share/doc/mbpfan
 	install $(BIN) $(DESTDIR)/usr/sbin
 	install -m644 $(CONF) $(DESTDIR)/etc
-	install -m644 $(DOC) $(DESTDIR)/usr/share/doc/mbpfan
-	install -d $(DESTDIR)/usr/share/man/man8
-	install -m644 $(MAN) $(DESTDIR)/usr/share/man/man8
+	install -d $(DESTDIR)/usr/man/man8
+	install -m644 $(MAN) $(DESTDIR)/usr/man/man8
 	@echo ""
 	@echo "******************"
 	@echo "INSTALL COMPLETED"
@@ -74,8 +71,6 @@
 	@echo "A configuration file has been copied (might overwrite existing file) to /etc/mbpfan.conf."
 	@echo "See README.md file to have mbpfan automatically started at system boot."
 	@echo ""
-	@echo "Please run the tests now with the command"
-	@echo "   sudo make tests"
 	@echo ""
 rebuild: clean all
 #rebuild is not entirely correct

