From 03b81aaf4051250f22cea80320feb0f748db9441 Mon Sep 17 00:00:00 2001
From: Dimitris Zlatanidis <d.zlatanidis@gmail.com>
Date: Thu, 15 Jan 2026 23:53:34 +0200
Subject: [PATCH] Fixed for aarch64 arch #201

Signed-off-by: Dimitris Zlatanidis <d.zlatanidis@gmail.com>
---
 slpkg/update_repositories.py | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/slpkg/update_repositories.py b/slpkg/update_repositories.py
index 7b2d6515..1349a0e0 100644
--- a/slpkg/update_repositories.py
+++ b/slpkg/update_repositories.py
@@ -39,7 +39,8 @@ class UpdateRepositories:  # pylint: disable=[R0902]
         self.red = config_load.red
         self.endc = config_load.endc
         self.lftp_mirror_options = config_load.lftp_mirror_options
-        self.is_64bit = config_load.is_64bit()
+        self.is_64bit: bool = config_load.is_64bit()
+        self.cpu_arch: str = config_load.cpu_arch
         self.view_process = ViewProcess()
 
         self.view = View(options)
@@ -179,7 +180,7 @@ class UpdateRepositories:  # pylint: disable=[R0902]
 
         if repo == self.repos.slack_repo_name:
             arch: str = ''
-            if self.is_64bit:
+            if self.is_64bit and self.cpu_arch != 'aarch64':
                 arch = '64'
             manifest = (f"{self.repos.repositories[repo]['mirror_packages']}slackware{arch}/"
                         f"{self.repos.repositories[repo]['manifest_bz2']}")
-- 
GitLab

